test/integration_test.dart

```diff
@@ -720,6 +728,15 @@ Future<void> main() async {
   });
 
   group('Home', () {
+    setUp(() {
+      when(mockAuthProvider.currentUser).thenAnswer((_) => Future.value(User(
```

Comment:
Add a test verifying that if the user sources are null, the sources page pops up?

Reply:
@AdrianMargineanu  can help with this?